rfcs/0138-rfc-repos.md

```diff
@@ -0,0 +1,106 @@
+
+The FCP must be initiated on a specific commit of the repository.
+When the FCP passes, the repositories contents are committed to the [rfcs](https://github.com/NixOS/rfcs) repository by the [RFC steering committee](https://github.com/NixOS/rfcs#rfc-steering-committee) with a commit that closes the original issue.
+
+## Workflow to get RFC updates
+To get updates for RFC's, instead of subscribing to the PR, one has to watch the repository.
+
+## Repository transfer
+
+Once the RFC is merged, the repository has to be transferred to the NixOS organization under https://github.com/NixOS/rfc-NUMBER. This is to ensure the discussions aren't lost in the future. The repository will then be archived.
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+
+# Drawbacks
+[drawbacks]: #drawbacks
+- There's no line-comment view anymore (the "Files changed" tab for PR's), where you can see the entire proposal and all (up-to-date) line comments.
+- Commenting on an RFC won't automatically subscribe you to updates anymore.
+- Less separation between RFC authors and reviewers with edit suggestions, making it harder to decide who may be a shepherd, since authors can't be shepherds.
+- Force pushes won't be visible anymore in the history.
+  - Force pushing shouldn't be necessary for RFCs, since they always get squash merged anyways
+    - With the previous approach, perhaps the main use case of force pushing was to reduce the PR items to be loaded,
+      since commits also count towards the number of items loaded by GitHub.
+      This is obviously not necessary anymore.
+  - It would be good to protect the main branch for active RFCs,
+    which could be done by requiring the RFC repo be moved to the NixOS organisation.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## Optional instead of required
+
+Instead of requiring this process, it could be opt-in for "bigger" RFCs.
+- (-) It's not clear how to decide whether an RFC should have a repository, there's no way to know how big discussions become in advance.
+
+## Fork branch instead of separate repository
+
+Instead of creating a new repository for each RFC, a new branch in a fork can be created instead.
+- (-) Means that a single GitHub user/organization can't have more than one RFC open at a time without mixing of issues/PRs occurring (since GitHub only supports having a single fork of a repository).
+  - (+) For time-distinct RFCs it can be worked around by [detaching the old fork](https://support.github.com/request/fork) and creating a new one
+- (+) Simplifies the RFC process, since one can just create a PR to upstream it
+  - (-) However this may again lead to the original problem of having too long PR discussions
+  - (-) It's also confusing about whether a PR is needed
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+
+# Future work
+[future]: #future-work
+
+- More RFC automation is possible in the future:
```

Comment:
In the spirit of automation, I think there will be a strong need of a template repository for RFCs, especially if a label architecture would be enforced as I mentioned in [this comment](https://github.com/NixOS/rfcs/pull/138/files#r1445091558).